source/support.tex

```diff
@@ -3395,11 +3395,10 @@
 The type \tcode{source_location} meets the
 \oldconcept{DefaultConstructible},
 \oldconcept{CopyConstructible},
-\oldconcept{Copy\-Assignable}, and
+\oldconcept{Copy\-Assignable},
+\oldconcept{Swappable}, and
 \oldconcept{Destructible}
-requirements\iref{utility.arg.requirements}.
-Lvalues of type \tcode{source_location}
-are swappable\iref{swappable.requirements}.
+requirements\iref{utility.arg.requirements, swappable.requirements}.
```

Comment:
```suggestion
requirements\iref{utility.arg.requirements,swappable.requirements}.
```
The convention.
`grep 'iref[^}]\+, ' *.tex` has no hits, whereas
`grep 'iref[^}]\+,' *.tex` has 190.

Reply:
OK :-)